### PR TITLE
Fixed the type hinting to make the tests pass.

### PR DESCRIPTION
**Pull request by fgm**

### CODE REVIEW

File: src/GraphQL/Execution/Processor.php [hunk 1]
```diff
@@ -69,14 +70,14 @@ protected function getResolveFunction(AbstractField $field) {
...
-  protected function resolveFieldValue(AbstractField $field, $contextValue, array $fields, array $args) {
+  protected function resolveFieldValue(FieldInterface $field, $contextValue, array $fields, array $args) {
     $resolveInfo = $this->createResolveInfo($field, $fields);
 
     if ($field instanceof Field) {
       if (($resolveFunction = $this->getResolveFunction($field)) && is_callable($resolveFunction)) {
         return $resolveFunction($contextValue, $args, $resolveInfo);
       }
-      else if ($propertyValue = TypeService::getPropertyValue($contextValue, $field->getName())) {
+      elseif ($propertyValue = TypeService::getPropertyValue($contextValue, $field->getName())) {
```
Comment (fubhy) [inline src/GraphQL/Execution/Processor.php:80]: Why this change? :)

Comment (fgm) [reply src/GraphQL/Execution/Processor.php:80]: Doesn't matter, just Drupal coding standards. The one important change is AbstractField to FieldInterface

**Comment by fgm** [comment]
No longer needed: the main point was already committed by @ohthehugemanatee without a PR.
